r2/r2/controllers/ipn.py

```diff
@@ -590,9 +590,9 @@ def finish(self, parameters, txn_id,
                         "supports the site and makes future development "
                         "possible. For example, one month of reddit gold "
                         "pays for 5 instance hours of reddit's servers.")
-            message += "\n\n" + strings.gold_benefits_msg
+            message += "\n\n" + _(strings.gold_benefits_msg)
```

Comment:
These are [already translated](https://github.com/reddit/reddit/blob/master/r2/r2/lib/strings.py#L124). Variables can't be translated. The way this works (I think) is the string inside `_()` lets [crowdin](https://www.reddit.com/r/i18n) know that it can be translated, so it enters the string into its database so that it can be translated by users.